app/assets/stylesheets/_global.scss

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
```

Comment:
A slightly nicer way to do this:

```css
html {
  box-sizing: border-box;
}

*,
*:before,
*:after {
  box-sizing: inherit;
}
```
https://css-tricks.com/box-sizing/#article-header-id-6